test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -455,6 +455,75 @@ public async Task ExtractContextIrrespectiveOfSamplingDecision(SamplingDecision
             }
         }
 
+        [Fact]
+        public async Task ExtractContextIrrespectiveOfTheFilterApplied()
+        {
+            try
+            {
+                var expectedTraceId = ActivityTraceId.CreateRandom();
+                var expectedParentSpanId = ActivitySpanId.CreateRandom();
+                var expectedTraceState = "rojo=1,congo=2";
+                var activityContext = new ActivityContext(expectedTraceId, expectedParentSpanId, ActivityTraceFlags.Recorded, expectedTraceState);
+                var expectedBaggage = Baggage.SetBaggage("key1", "value1").SetBaggage("key2", "value2");
+                Sdk.SetDefaultTextMapPropagator(new ExtractOnlyPropagator(activityContext, expectedBaggage));
```

Comment:
nit - for a diff pr: The name `ExtractOnlyPropagator` is confusing here. It could be named something like `TestPropagator`.